packages/api-server/src/modules/oauth.ts

```diff
@@ -29,8 +39,8 @@ export class AuthController {
         data: {
           client_id:
             "587163973906-od2u5uaop9b2u6ero5ltl342hh38frth.apps.googleusercontent.com",
```

Comment:
Tomorrow I'll look into the proper way to inject both client_id and client_secret at build time.  Meanwhile, can you add these two props into `GetTokenOpts`?

Reply:
Sure thing. Added in RefreshTokenOpts as well, required for refreshing token.